src/ucp/api/ucp.h

```diff
@@ -844,7 +846,7 @@ typedef struct ucp_ep_params {
      * if the @ref UCP_EP_PARAMS_FLAGS_CLIENT_SERVER flag is set, this address
      * is mandatory for filling (along with its corresponding bit in the
      * field_mask - @ref UCP_EP_PARAM_FIELD_SOCK_ADDR) and should be obtained
-     * from the user.
+     * from the user. This field cannot be changed by @ref ucp_ep_modify_nb.
      */
     ucs_sock_addr_t         sockaddr;
```

Comment:
do we have the check and error in ucp_ep_modify_nb ?

Reply:
Yes, see https://github.com/openucx/ucx/pull/2130/files#diff-8fd5f37669b4533d1e9389ff6839dcccR400